examples/streaming/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Snap Plugin Go Library: Stream Collector Plugin Example
+Here you will find an example plugin that covers the basics for writing a stream collector plugin.
+
+## Plugin Naming, Files, and Directory
```

Comment:
If the naming for stream plugins is the same as the traditional plugins, maybe just a reference to plugin naming documentation is enough for this section.